composer.json

```diff
@@ -36,33 +36,32 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "composer-runtime-api": "^2.0",
-        "coduo/php-matcher": "^6.0.14",
         "composer/package-versions-deprecated": "^1.8",
         "contao/imagine-svg": "^1.0",
-        "doctrine/annotations": "^1.2 || ^2.0",
-        "doctrine/cache": "^1.0.1",
-        "doctrine/collections": "^1.0 || ^2.0",
-        "doctrine/data-fixtures": "^1.3.3",
-        "doctrine/dbal": "^2.13 || ^3.0",
-        "doctrine/doctrine-bundle": "^2.4",
-        "doctrine/doctrine-fixtures-bundle": "^3.3 || ^4.0",
+        "doctrine/annotations": "^1.14 || ^2.0",
+        "doctrine/cache": "^1.12.1",
+        "doctrine/collections": "^1.6 || ^2.0",
+        "doctrine/data-fixtures": "^1.4",
+        "doctrine/dbal": "^3.3",
+        "doctrine/doctrine-bundle": "^2.6",
+        "doctrine/doctrine-fixtures-bundle": "^3.4 || ^4.0",
         "doctrine/inflector": "^1.4.1 || ^2.0.1",
         "doctrine/orm": "^2.13",
-        "doctrine/persistence": "^2.0 || ^3.0",
-        "doctrine/phpcr-bundle": "^2.2 || ^3.0",
+        "doctrine/persistence": "^3.1",
+        "doctrine/phpcr-bundle": "^2.4 || ^3.0",
         "dragonmantank/cron-expression": "^1.1 || ^2.0 || ^3.0",
         "friendsofphp/proxy-manager-lts": "^1.0",
-        "friendsofsymfony/http-cache": "^2.10 || ^3.0",
-        "friendsofsymfony/http-cache-bundle": "^2.10.1 || ^3.0",
-        "friendsofsymfony/jsrouting-bundle": "^2.7 || ^3.0",
-        "friendsofsymfony/rest-bundle": "^3.1",
+        "friendsofsymfony/http-cache": "^2.13 || ^3.0",
+        "friendsofsymfony/http-cache-bundle": "^2.12 || ^3.0",
+        "friendsofsymfony/jsrouting-bundle": "^3.0",
+        "friendsofsymfony/rest-bundle": "^3.2",
         "gedmo/doctrine-extensions": "^3.0.4",
         "guzzlehttp/promises": "^1.0 || ^2.0",
         "handcraftedinthealps/goodby-csv": "^1.4",
         "handcraftedinthealps/rest-routing-bundle": "^1.0",
         "imagine/imagine": "^1.0",
         "jackalope/jackalope": "^1.4.5 || ^2.0",
-        "jms/serializer": "^3.16",
+        "jms/serializer": "^3.19",
```

Comment:
I increased the version to the one which was installed when using `--prefer-lowest` e.g. doctrine/dbal 2 is not longer supported by our dependencies so we can increase it also in our own composer.json.